domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/showcondition/ShowCondition.java

```diff
@@ -1,42 +1,63 @@
 package uk.gov.hmcts.ccd.definition.store.domain.showcondition;
 
+import lombok.ToString;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@ToString
 public class ShowCondition {
 
-    private String showConditionExpression;
+    private final StringBuilder showConditionExprBuilder = new StringBuilder();
```

Comment:
why we need this StringBuilder inside the ShowCondition? Seems to me it's only needed because of the particular way the ShowCondition.Builder has been implemented.
The show condition, even if it's a complex one now, could remain an immutable String as it was before. 
What you could do is to modify the Builder to pass the final show condition string at build time. See my comment below

Reply:
Done